README.md

```diff
@@ -1,17 +1,139 @@
-# Registers Ruby Client
+# GOV.UK Registers Ruby Client
+
+This documentation is for developers interested in using this Ruby client to integrate their service with GOV.UK Registers. 
+
 ## Installation
 In your Gemfile add:
 ```
 gem 'registers-ruby-client', git: 'https://github.com/openregister/registers-ruby-client.git'
 ```
 ## Usage
+
+_Note: examples use the `country` register_
+
 ```
 require 'registers_client'
 
 registers_client = RegistersClient::RegistersClientManager.new({ cache_duration: 3600 })
+
+
 register_data = registers_client.get_register 'country', 'beta'
-register_data.get_records.first[:item]
- => {"citizen-names"=>"Soviet citizen", "country"=>"SU", "end-date"=>"1991-12-25", "name"=>"USSR", "official-name"=>"Union of Soviet Socialist Republics"} 
- ```
+```
+
+### Methods
+
+#### `get_entries`
+
+Example usage:
+
+`register_data.get_entries.first[:item]`
+
+Expected output:
+
+```
+{"citizen-names"=>"Soviet citizen", "country"=>"SU", "end-date"=>"1991-12-25", "name"=>"USSR", "official-name"=>"Union of Soviet Socialist Republics"}
+get_records
+```
+
+#### `get_records`
+
+Example usage:
+
+`register_data.get_records.first[:item]`
+
+Expected output:
+
+```
+{"citizen-names"=>"Soviet citizen", "country"=>"SU", "end-date"=>"1991-12-25", "name"=>"USSR", "official-name"=>"Union of Soviet Socialist Republics"}
+```
+
+#### `get_metadata_records`
+
+Example usage:
+
+`register_data.get_metadata_records.first[:item]`
+
+Expected output:
+
+`{"name"=>"country"}`
+
+#### `get_field_definitions`
+
+Example usage:
+
+`register_data.get_field_definitions.first[:item]`
+
+Expected output:
+
+```
+{"cardinality"=>"1", "datatype"=>"string", "field"=>"country", "phase"=>"beta", "register"=>"country", "text"=>"The country's 2-letter ISO 3166-2 alpha2 code."}
+```
+
+#### `get_register_definition`
+
+Example usage:
+
+`register_data.get_register_definition.to_json`
```

Comment:
I would unescape the JSON to make it more readable.

Reply:
Done in f4eaa04ac550a00a8020d61b7f61baef2158aefd